manifests/install.pp

```diff
@@ -8,4 +8,17 @@
   package{'ferm':
     ensure => 'latest',
   }
+
+  case $facts['os']['family'] {
+    'RedHat': {
+      if scanf($facts['os']['release']['major'], '%i')[0] <= 6 {
```

Comment:
interesting to use `scanf()` and not `versioncmp()`. The latter is more common, but `scanf` is cooler. :+1:

Reply:
Could we go with `versioncmp` though? :)

A single `if` instead of a `case` and an `if` would probably be simpler too.